### PR TITLE
[Issue 30]: Init project

### PR DESCRIPTION
**Pull request by khoadev2512**
link issue: https://github.com/HopeeSolutionsRuby/demo-app/issues/30

For first clone: 
  Create file .env and assign: 
      MYSQL_PASSWORD=SOMETHING 
  Create database: 
      rails db:create

### CODE REVIEW

**Comment by hothanhdatsd** [comment]
review done

**Review by hothanhdatsd — APPROVED** [review]

**Review by minhnguyennn — APPROVED** [review]

**Review by minhnguyennn — APPROVED** [review]

**Review by PhiH-XDev — APPROVED** [review]

**Comment by minhnguyennn** [comment]
I approved this pull request.

File: config/credentials.yml.enc [hunk 1]
```diff
@@ -0,0 +1 @@
+BKK4YKS5OWvAJL4k2t5ga31GeKUa6KVtnYS0bRLlf0Qtlp16naALJOQZRjuRdDwhdekohJykCt8cw+23qS1Zidv/BabrOIYzhl/xOrsgLPx/OsNwix6FslSxPCWs3Vn0+/BospnpOJ/K2U9YTywqImkwG5RSxWGZt6xi7osqxhgOW3+5IBSTyuNnaNUHbEqIMeGpghPN9st1vO+3iBn7AJt5KMh0xF5j/v7JesQ8EjrTGqSx+nZICI7aC+d3dn/6uMcXH/aJ2z6/4VnGLTmjXjaipjlImnOIRjXwUG4h40K1NiGsu31cKRnBVVuN+SBNKX2qW9jHNA2bRomP+s/7fhXgAegLYseIVL1wFXi9gFCZAg2EZrpnAcaXP71XzQLXWFdpbqiVBNmmqQeI9N9UrlYSm7Pv--V7sH48XGtKW7M2Bk--ZsfYNof1Cntj0KMqHrye+w==
```
Comment (KhoaNDAHopee) [inline config/credentials.yml.enc]: Should be empty line

**Review by KhoaNDAHopee — CHANGES_REQUESTED** [review]

File: config/credentials.yml.enc (hunk 1 above)
Comment (khoadev2512) [reply config/credentials.yml.enc]: Fixed

**Review by hothanhdatsd — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
-New project
\ No newline at end of file
+# Line CRM API (Rails API)
```
Comment (KhoaNDAHopee) [inline README.md]: project name?

File: README.md [hunk 3]
```diff
@@ -1 +1,36 @@
...
+
+# 1. Prerequisites
+- Ruby 3.2.0
+- Rails 7.0.4.3
```
Comment (KhoaNDAHopee) [inline README.md]: Sure?

File: README.md [hunk 4]
```diff
@@ -1 +1,36 @@
...
+
+# 2. Create db schema
+- Run
+'''
+    rails db:create
+'''
+
+# 3. Configure mysql db connection
```
Comment (KhoaNDAHopee) [inline README.md:17]: Should be do this step first, before ``Create db schema``

**Review by KhoaNDAHopee — CHANGES_REQUESTED** [review]
Please check comments

**Review by KhoaNDAHopee — APPROVED** [review]
LGTM!

**Pull request merged**